src/rpc/signaling-exchange.ts

```diff
@@ -365,30 +366,30 @@ const iceCandidateFromProto = (i: ICECandidate) => {
   const candidate: RTCIceCandidateInit = {
     candidate: i.candidate,
   };
-  if (i.sdpMid) {
+  if (i.sdpMid !== undefined) {
```

Comment:
This is the fix. We we were excluding the right properties because some of them were falsey